src/Compilers/VisualBasic/Portable/Symbols/Retargeting/RetargetingSymbolTranslator.vb

```diff
@@ -149,7 +149,11 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols.Retargeting
             Private Function RetargetNamedTypeDefinition(type As NamedTypeSymbol, options As RetargetOptions) As NamedTypeSymbol
                 Debug.Assert(type Is type.OriginalDefinition)
 
-                ' Before we do anything else, check if we need to do special retargeting
+                If (type.IsTupleType) Then
+                    Return DirectCast(type, TupleTypeSymbol).WithUnderlyingType(Retarget(type.TupleUnderlyingType, options))
```

Comment:
Thanks for pointing this out. I was looking at the PR that introduced the logic here first, but not the most recent version of the implementation.

There is one test that manages to produce a retargeted underlying type that isn't good: `NoPiaEmbedTypes.ErrorType_Tuple`. I'll add this check, but that test will only be ported later.